example/package.json

```diff
@@ -11,8 +11,8 @@
   },
   "dependencies": {
     "react": "16.13.1",
-    "react-native": "0.63.0",
-    "react-native-device-info": "github:react-native-community/react-native-device-info"
+    "react-native": "^0.62.2",
+    "react-native-windows": "^0.62.5"
```

Comment:
actually (sorry :sweat_smile: ) I don't think it is valid to remove the react-native-device-info here?

And (confirming) there is no way to use react-native@^0.63.2 (current stable) with react-native-windows at the moment? I'd prefer to stay on current stable for the example if possible (I have a script that does it automatically actually, so the version needs to be pinned in the script if 0.63.x won't work)

Reply:
I tried to use combination "react-native": "0.63.0" & "react-native-windows": "^0.62.5" and i got build error:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	C1083	Cannot open include file: 'winrt/Microsoft.ReactNative.h': No such file or directory	Microsoft.ReactNative	C:\Users\tepaa\Developer\next\forks\rn-device-info\example\node_modules\react-native-windows\Microsoft.ReactNative\HResult.h	6	


Severity	Code	Description	Project	File	Line	Suppression State
Error	C1083	Cannot open source file: 'C:\Users\tepaa\Developer\next\forks\rn-device-info\example\node_modules\react-native-windows\build\x64\Debug\ReactCommon\react-native-patched\ReactCommon\callinvoker\ReactCommon\BridgeJSCallInvoker.cpp': No such file or directory	ReactCommon	C:\Users\tepaa\Developer\next\forks\rn-device-info\example\node_modules\react-native-windows\ReactCommon\c1xx	1
```

I will make new PR when react-native-windows 0.63.x is "ready".